File: lista2sala/4.py
a=int(input("digite o valor do 1º lado do triangulo:"))
b=int(input("digite o valor do 2º lado do triangulo:"))
c=int(input("digite o valor do 3º lado do triangulo:"))
triangulo=True
if a<b+c and b<a+c and c<a+b:
    triangulo=True
else:
    triangulo=False
if triangulo==True:
    if (a==b) and (b==c):
        print("o triangulo é equilátero")
    elif (a==b) or (a==c) or (b==c):
        print("o triangulo é isósceles")
    else:
        print("o triangulo é escaleno")
if triangulo==False:
    print("o valor dado não corresponde a um triangulo")

File: listas/lista2sala/4.c
#include <stdio.h>
#include <stdbool.h>
int main(){
    float a, b, c, t;
    printf("entre com os valores dos lados dos triangulos:");
    scanf("%d%d%d", &a, &b, &c);
    if ((a<b+c)&&(b<a+c)&&(c<a+b)){
        t=true;
    }
    else{
        t=false;
    }
    if (t==true){
        if((a==b)&& (b==c))
        printf("o triangulo é equilatero!");
    }
        if ((a==b) || (a==c)|| (b==c)){
            printf("o triangulo é isosceles!");
        }
        else{
            printf("o triangulo e escaleno");
        }
    if (t==false){
        printf("o valor dado nao corresponde a um triangulo");
    }
}